### ISSUE TITLE
let's use issue as our Q&A space!

### ISSUE THREAD

**Issue by emma-data-works** [emma-data-works/ds-class-intro#1]
how do we submit HW1 @emma-data-works

**Comment by XXXVivilovesBoba** [comment]
Can I ask a stupid question?

num_list = [2,4,6,8]
for position, number in enumerate(num_list):
    print('value at index {0} is {1}'.format(position,number))

For the code listed above, I understand {0} indicates the position, but why do we put 1 in the second {} as {1}? I tried other numbers and it did not work.

**Comment by XXXVivilovesBoba** [comment]
@emma-data-works Is that because Python just assumes the position starts with 0 and the nature number starts with 1?

**Comment by emma-data-works** [comment]
> @emma-data-works Is that because Python just assumes the position starts with 0 and the nature number starts with 1?

Yes, basically if you have more than one values to feed into the formatted/template string, they will use the default position indices like 0,1,2,3,... As we have `position` and `number` to fill in the blanks in the format string, we can use `0` and `1` to tell the program where to put these values. 
If you change the value to other numbers than `0` and `1`, since you only give two values in the `format()`, it will error out since python can not find the value with such index.


If we have 4 values, we can have some things like this:
```
print('I'd like to print out {0}, {1}, {2}, {3} in this sentence'.format(word1, word2, word3, word4))
```

Hope this answers your question!
